cypress/integration/visitations.ts

```diff
@@ -0,0 +1,116 @@
+import faker from 'faker';
+
+faker.seed(1);
```

Comment:
What is this used for. I haven't used this seed before just wondering what it is doing.

Reply:
It uses the same random values for the test depending on the seed given.
It is here to avoid false positives when running E2E